include/cute/atom/copy_traits_xe.hpp

```diff
@@ -166,9 +166,15 @@ struct XE_2D_LD_Unpack {
   CUTE_HOST_DEVICE friend constexpr void
   copy_unpack(Traits_LD_t const &traits, Tensor<TS, SLayout> const &src,
               Tensor<TD, DLayout> &dst) {
-    static_assert(is_rmem<TD>::value);
-
     using dtype = typename Tensor<TD, DLayout>::value_type;
+    constexpr int dtype_size = sizeof(dtype);
+    constexpr int bits_in_byte = 8;
```

Comment:
Cutlass provides `cutlass::sizeof_bits<dtype>` for this

Reply:
done